### ISSUE TITLE
关于PPYOLOE_S使用自动压缩方法在训练过程中的loss值的问题

### ISSUE THREAD

**Issue by Nirvana93** [PaddlePaddle/PaddleSlim#1676]
实验通过[官方示例](https://github.com/PaddlePaddle/PaddleSlim/tree/develop/example/auto_compression/detection)进行，
相关配置设置如下：
```
Global:
  exclude_nms: True
  arch: PPYOLOE    

Distillation:
  alpha: 1.0
  loss: soft_label

QuantAware:
  onnx_format: true
  use_pact: true
  activation_quantize_type: 'moving_average_abs_max'
  quantize_op_types:
  - conv2d
  - depthwise_conv2d

TrainConfig:
  train_iter: 5000
  eval_iter: 1000
  learning_rate:  
    type: CosineAnnealingDecay
    learning_rate: 0.00003
    T_max: 6000
  optimizer_builder:
    optimizer: 
      type: SGD
    weight_decay: 4.0e-05
```

模型训练时的loss如下，请问这种大小的loss数值是正常的吗
```
2023-03-04 15:22:00,387-INFO: Total iter: 0, epoch: 0, batch: 0, loss: [13.207467]soft_label: [13.207467] 
2023-03-04 15:22:03,056-INFO: Total iter: 10, epoch: 0, batch: 10, loss: [12.428257]soft_label: [12.428257] 
2023-03-04 15:22:05,751-INFO: Total iter: 20, epoch: 0, batch: 20, loss: [12.663028]soft_label: [12.663028] 
2023-03-04 15:22:08,447-INFO: Total iter: 30, epoch: 0, batch: 30, loss: [14.896362]soft_label: [14.896362] 
2023-03-04 15:22:11,144-INFO: Total iter: 40, epoch: 0, batch: 40, loss: [12.247461]soft_label: [12.247461] 
2023-03-04 15:22:13,840-INFO: Total iter: 50, epoch: 0, batch: 50, loss: [12.451904]soft_label: [12.451904] 
2023-03-04 15:22:16,536-INFO: Total iter: 60, epoch: 0, batch: 60, loss: [12.284585]soft_label: [12.284585] 
2023-03-04 15:22:19,237-INFO: Total iter: 70, epoch: 0, batch: 70, loss: [12.151497]soft_label: [12.151497] 
2023-03-04 15:22:21,940-INFO: Total iter: 80, epoch: 0, batch: 80, loss: [12.264659]soft_label: [12.264659] 
2023-03-04 15:22:24,642-INFO: Total iter: 90, epoch: 0, batch: 90, loss: [12.188266]soft_label: [12.188266] 
...
2023-03-04 15:45:20,086-INFO: Total iter: 4959, epoch: 10, batch: 280, loss: [12.240063]soft_label: [12.240063] 
2023-03-04 15:45:22,784-INFO: Total iter: 4969, epoch: 10, batch: 290, loss: [12.523859]soft_label: [12.523859] 
2023-03-04 15:45:25,482-INFO: Total iter: 4979, epoch: 10, batch: 300, loss: [12.421528]soft_label: [12.421528] 
2023-03-04 15:45:28,180-INFO: Total iter: 4989, epoch: 10, batch: 310, loss: [11.971611]soft_label: [11.971611] 
2023-03-04 15:45:30,950-INFO: Total iter: 4999, epoch: 10, batch: 320, loss: [12.2105465]soft_label: [12.2105465] 
```

**Comment by zzjjay** [comment]
这个loss数值是正常的。
自动压缩实验都是通过配置知识蒸馏进行压缩训练，PPYOLOE_S 的示例设置了soft label 蒸馏，所以数值会和原始有标签 loss 相差较大。

**Comment by Nirvana93** [comment]
> 这个loss数值是正常的。 自动压缩实验都是通过配置知识蒸馏进行压缩训练，PPYOLOE_S 的示例设置了soft label 蒸馏，所以数值会和原始有标签 loss 相差较大。

好的谢谢
